### ISSUE TITLE
Networks policies that are completely open are ignored

### ISSUE THREAD

**Issue by lhankinsgdl** [runoncloud/kubectl-np-viewer#2]
A network policy that looks like:

```
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: wide-open-network-policy
spec:
  podSelector:
    matchLabels:
      app: some-app
  policyTypes:
    - Ingress
    - Egress
  ingress:
    - {}
  egress:
    - {}
```

...will not get printed.  It seems that, since there are no peers, there is nothing in each of the ingress or egress loops that will catch it and print it out.

**Comment by lhankinsgdl** [comment]
Actually, so are ones that are completely closed, like a default-deny:

```
# Default Deny
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: default-deny
spec:
  podSelector: {}
  policyTypes:
    - Ingress
    - Egress
```

**Comment by jeanpbond** [comment]
Thank you for your feedback @lhankinsgdl! This will be fixed in the next release planned for tomorrow.

**Comment by jeanpbond** [comment]
Fixed in v1.0.6.
